docs/modules/ROOT/pages/runbooks/APPUiOCloudInvoiceGenerationFailed.adoc

```diff
@@ -1,18 +1,67 @@
-= Alert rule: APPUiOCloudReportingDatabaseBackfillingFailed
+= Alert rule: APPUiOCloudInvoiceGenerationFailed
 
 include::partial$runbooks/contribution_note.adoc[]
 
 == icon:glasses[] Overview
 
-APPUiO Cloud Reporting failed while trying to generate Invoices.
+APPUiO Cloud Reporting failed while trying to generate invoices.
 Invoices might be incomplete or missing.
+Remove any obstacles and rerun the job.
 
-Deleting the failed job will clear this alert.
+[IMPORTANT]
+====
+The job isn't idempotent.
+Rerunning the job, can leave duplicates which need to be cleaned up.
+====
 
 == icon:bug[] Steps for debugging
 
-include::partial$runbooks/common_debugging_steps.adoc[]
+. Identify source of failure
+* Check for missing data indicated by the alert rule xref:runbooks/APPUiOCloudReportingMissingData.adoc[APPUiOCloudReportingMissingData].
+* Check logs for other issues
```

Comment:
Might be nice to list other issues that can occur (not sure we're explicitly aware of anything other than missing data which could cause invoice generation to fail)